src/edu/ucsb/nceas/metacat/dataone/v1/MNodeService.java

```diff
@@ -98,7 +98,7 @@ public static MNodeService getInstance(HttpServletRequest request) {
      * Constructor, private for singleton access
      */
     private MNodeService(HttpServletRequest request) {
-        logMetacat = Logger.getLogger(MNodeService.class);
+        logMetacat = LogFactory.getLog(MNodeService.class);
```

Comment:
```suggestion
```

Reply:
See the commit 70db73cab0ce506277782cd20f081b0dabcd9c8a.